lib/actions/read.js

```diff
@@ -162,7 +162,7 @@ module.exports.getMetaModel = async function getMetaModel(cfg) {
     out: {}
   };
 
-  if (cfg.emitBehavior === 'emitBatch') {
+  if (cfg.emitAll === 'emitBatch') {
```

Comment:
```suggestion
  if (emitBehavior === 'emitBatch') {
```

Reply:
Just to make it a little bit more clear

Reply:
This won't work. 'emitBehavior' is out of scope